Server/Auth/Auth.Application/Validation/UserCredentialsValidator.cs

```diff
@@ -0,0 +1,13 @@
+using FluentValidation;
+using ModularHouse.Server.Auth.Application.Dto;
+
+namespace ModularHouse.Server.Auth.Application.Validation;
+
+public class UserCredentialsValidator : AbstractValidator<UserCredentials>
+{
+    public UserCredentialsValidator()
+    {
+        RuleFor(credentials => credentials.Email).EmailAddress();
+        RuleFor(credentials => credentials.Password).Password();
```

Comment:
`.Password()` is the correct naming in this situation. As you can see, the `.EmailAddress()` method is default method out of the box of `FluentValidator`. It's not necessary to add `Validate` prefix, because method can be used only for validation.